api/base/views.py

```diff
@@ -405,14 +405,18 @@ def root(request, format=None, **kwargs):
         current_user = UserSerializer(user, context={'request': request}).data
     else:
         current_user = None
+
     flags = [name for name in Flag.objects.values_list('name', flat=True) if flag_is_active(request, name)]
+    samples = [name for name in Sample.objects.values_list('name', flat=True) if sample_is_active(name)]
+    switches = list(Switch.objects.filter(active=True).values_list('name', flat=True))
+
     kwargs = request.parser_context['kwargs']
     return_val = {
         'meta': {
             'message': 'Welcome to the OSF API.',
             'version': request.version,
             'current_user': current_user,
-            'active_flags': flags,
+            'active_flags': flags + samples + switches,
```

Comment:
Doesn't need to be changed in this PR, but would be nice to version this and update the key to be `active_features`.